### ISSUE TITLE
Foreslår ny term Knoper (og stikk?)

### ISSUE THREAD

**Issue by Hypsibius** [realfagstermer/realfagstermer#484]
jf https://snl.no/stikk_og_knoper NN knopar EN knots

belegg: http://bibsys-almaprimo.hosted.exlibrisgroup.com/UBB:default_scope:BIBSYS_ILS71512300810002201

Vi har frå før knuteteori BF knuter, men burde denne rettast til knuteteori BF knuter (matematikk), og så kunne denne få BF knuter (tauverk)? @mittinatten ?

**Comment by mittinatten** [comment]
Den er grei i prinsipp, hvis det gir mening å ha fagspesifisering i BF-termene. Burde og ha med en SO mellom de.

**Comment by Hypsibius** [comment]
Ja, var litt usikker på fagspesifiseringen der. SO nødvendig. Men synest merkeleg om 'knuter' skal vise kun til knuteteori, og ikkje til knop og denslags, som folk gjerne først tenker på ;-)

Frøvri: Kan det heite Knoper og stikk (evt omvendt) eller bør 'knoper' vere hovudinnførsel og så stikk BF-term? Eller, i så fall er vel 'knuter' den termen som dekkjer begge?

**Comment by danmichaelo** [comment]
Hadde det funka med noe sånt?

```
Knuteteori
BF Knuter (matematikk)
SO Knuter
```

```
Knuter
NO Knuter på tauverk og annet materiale. For matematiske knuter, se Knuteteori
BF Knoper
BF Stikk
SO Knuteteori
```

**Comment by Hypsibius** [comment]
For meg, så

**Comment by bibliomari** [comment]
Gjort.

**Comment by danmichaelo** [comment]
@Hypsibius , vil du legge dette emnet på boka som ble nevnt innledningsvis? Nå kommer Knuter opp i lista over Realfagstermer som ikke er i bruk.
